### PR TITLE
Remove :expr from placeholders

### PR DESCRIPTION
**Pull request by davidlattimore**
Reasoning discussed at https://github.com/rust-analyzer/rust-analyzer/issues/3186#issuecomment-644069324

### CODE REVIEW

File: crates/ra_ide/src/ssr.rs [hunk 1]
```diff
@@ -27,11 +27,11 @@ impl std::error::Error for SsrError {}
 //
 // Search and replace with named wildcards that will match any expression.
 // The syntax for a structural search replace command is `<search_pattern> ==>> <replace_pattern>`.
-// A `$<name>:expr` placeholder in the search pattern will match any expression and `$<name>` will reference it in the replacement.
+// A `$<name>` placeholder in the search pattern will match any AST node and `$<name>` will reference it in the replacement.
 // Available via the command `rust-analyzer.ssr`.
 //
 // ```rust
-// // Using structural search replace command [foo($a:expr, $b:expr) ==>> ($a).foo($b)]
+// // Using structural search replace command [foo($, $b) ==>> ($a).foo($b)]
```
Comment (bjorn3) [inline crates/ra_ide/src/ssr.rs:34]: *$a

Comment (davidlattimore) [reply crates/ra_ide/src/ssr.rs]: Thanks. I amended the commit and force pushed. Hopefully that was the right thing to do. I've not done a lot of PRs on github, so not really sure of etiquette.

Comment (matklad) [reply crates/ra_ide/src/ssr.rs]: We generally are pretty lax as far as workflow and code style goes, "best practicies" are being documented here:

https://github.com/rust-analyzer/rust-analyzer/blob/master/docs/dev/README.md#code-style--review-process

**Pull request merged**